### PR TITLE
Update nginx.md

### PR DESCRIPTION
**Pull request by alexh-name**
Configuration regarding PHP(-FPM) was outdated and the path for webroot wasn't correct.

### CODE REVIEW

File: remote-access/web-server/nginx.md [hunk 1]
```diff
@@ -106,7 +101,7 @@ sudo /etc/init.d/nginx reload
 Rename `index.html` to `index.php`:
 
 ```bash
-cd /usr/share/nginx/www/
+cd /var/www/html/
 sudo mv index.html index.php
```
Comment (JamesH65) [inline remote-access/web-server/nginx.md]: I believe that the file in /var/www/html is now called index.nginx-debian.html so this line should be updated as such.

Comment (alexh-name) [inline remote-access/web-server/nginx.md]: You were right. See 308939ef821cc62aa909d7500640047191e650b5.

**Comment by alexh-name** [comment]
Made the change in 308939ef821cc62aa909d7500640047191e650b5.

**Comment by JamesH65** [comment]
Pushed to copy edit for final review and merge.

**Comment by JamesH65** [comment]
fixes #370

**Comment by LornaLynch** [comment]
Changed sections look fine from copy POV. Merging.

**Pull request merged**